emmtyper/0.2.0/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM mambaorg/micromamba:1.2.0 as app
+
+# build and run as root users since micromamba image has 'mambauser' set as the $USER
+USER root
+# set workdir to default for building
+WORKDIR /
+
+ARG EMMTYPER_VER="0.2.0"
+
+LABEL base.image="mambaorg/micromamba:1.2.0"
+LABEL dockerfile.version="1"
+LABEL software="emmtyper"
+LABEL software.version=${EMMTYPER_VER}
+LABEL description="Conda environment for emmtyper. emmtyper is a command line tool for emm-typing of Streptococcus pyogenes using a de novo or complete assembly."
+LABEL website="https://github.com/MDU-PHL/emmtyper"
+LABEL license="GNU General Public License v3.0"
+LABEL license.url="https://github.com/MDU-PHL/emmtyper/blob/master/LICENSE"
+LABEL maintainer="Henry Kunerth"
+LABEL maintainer.email="henrykunerth@gmail.com"
+
+# install dependencies; cleanup apt garbage
+RUN apt-get update && apt-get install -y --no-install-recommends \
+ wget && \
+ apt-get autoclean && rm -rf /var/lib/apt/lists/*
+ 
+# set the environment
+ENV PATH="$PATH" \
+ LC_ALL=C.UTF-8
+
+#install emmtyper and dependencies
+RUN micromamba create -n emmtyper && \
+    eval "$(micromamba shell hook --shell=bash)" && \
+    micromamba activate emmtyper && \
+    micromamba install --yes --name emmtyper -c conda-forge -c bioconda -c defaults emmtyper && \
+    micromamba clean --all --yes

```

Comment:
Could you specify which emmtyper version you're using? It helps with the longevity of the Dockerfiles and images.

An example you could use is [Mykrobe](https://github.com/StaPH-B/docker-builds/blob/master/mykrobe/0.12.1/Dockerfile#L64-L69)

```
RUN micromamba create -n mykrobe -y -c conda-forge -c bioconda -c defaults \
 mykrobe=${MYKROBE_VER} \
 python \ 
 pip \
 pandas && \
 micromamba clean -a -y
 ```

Reply:
Makes sense - I followed the Mykrobe approach in the updated version.